gradebook/src/GradeBook/Statistics.cs

```diff
@@ -0,0 +1,9 @@
+namespace GradeBook
+{
+    public class Statistics
+    {
+        public double Avg;
```

Comment:
We generally prefer properties to fields.  Changing a property to do additional work (e.g. caching, calculations) is a non-breaking change, but changing a field to do the same is a breaking change.